website_event_sale_b2x_alt_price/readme/ROADMAP.md

```diff
@@ -0,0 +1,7 @@
+- When the public pricelist is set to display discounts in customer's
+  face and the website is set to display prices tax included, the
+  discounted price will appear always as tax excluded and produce weird
+  scenarios. This is Odoo's bug and not this module's. It has been
+  notified as OPW-2518694 but it doesn't look like it's going to be
+  fixed any soon. In v16, event price discounts from pricelist don't
+  even work and for v17 there's an ongoing refactoring work.
```

Comment:
Did you check this?

Reply:
Is correct

![image](https://github.com/user-attachments/assets/ec9b981c-1331-472c-9987-d0e8ea4f6fa5)

Reply:
Can you test it with a discount?

Reply:
Done!